cmake/Modules/FindMKLDNN.cmake

```diff
@@ -42,18 +42,14 @@ IF(NOT MKLDNN_FOUND)
         list(APPEND DNNL_MAKE_COMMAND "--" "-l" "$ENV{MAX_JOBS}")
       endif()
     endif()
-    if(LINUX)
-      set(DNNL_CXX_FLAGS "-DCMAKE_CXX_FLAGS=-fpreview-breaking-changes")
-    else()
-      set(DNNL_CXX_FLAGS "")
-    endif()
+    set(DNNL_CXX_FLAGS ${CMAKE_CXX_FLAGS} ${XPU_DEVICE_CXX_FLAGS})
```

Comment:
In addition, it changes the `CMAKE_CXX_FLAGS` for both Linux and Windows. Pls. pass the E2E test cases in torch-xpu-ops.

Reply:
It couldn't change `CMAKE_CXX_FLAGS` on Windows because `XPU_DEVICE_CXX_FLAGS` is empty on Windows.